common/src/api/external/mod.rs

```diff
@@ -1046,6 +1046,7 @@ pub enum ResourceType {
     Oximeter,
     MetricProducer,
     RoleBuiltin,
+    TargetRelease,
```

Comment:
Where does this get used?  Is it that the `authz_resource!` macro expected it?

I wouldn't really expect this to be needed because I think it's mainly used for generating 404s and you can't get a 404 on a target release.

Reply:
Thank you, fixed and replaced with a synthetic resource in 5fe4bb31b.